Problem3/test/problem3/Problem3Test.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem3;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Random;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Evan
+ */
+public class Problem3Test {
+    
+    private static Person[] personList;
+    
+    @Before
+    public void setUp()
+    {
+        personList = new Person[10]; //Create an array of Person objects
+        personList[0] = new Person("56", "person1", 16);
+        personList[1] = new Person("76", "person2", 16);
+        personList[2] = new Person("42", "person3", 16);
+        personList[3] = new Person("12", "person4", 16);
+        personList[4] = new Person("46", "person5", 16);
+        personList[5] = new Person("60", "person6", 16);
+        personList[6] = new Person("72", "person7", 16);
+        personList[7] = new Person("38", "person8", 16);
+        personList[8] = new Person("87", "person9", 16);
+        personList[9] = new Person("82", "person10", 16);
+    }
+
+    /**
+     * Test of insertionSort method, of class Problem3.
+     */
+    @Test
+    public void testInsertionSort()
```

Comment:
Your testing should include edge cases like empty arrays and arrays of a single element.